instrumentation/grpc-1.5/src/main/java/io/opentelemetry/auto/instrumentation/grpc/client/GrpcClientBuilderInstrumentation.java

```diff
@@ -58,6 +58,8 @@ public GrpcClientBuilderInstrumentation() {
       packageName + ".TracingClientInterceptor$TracingClientCall",
       packageName + ".TracingClientInterceptor$TracingClientCallListener",
       "io.opentelemetry.auto.instrumentation.grpc.common.GrpcHelper",
+      // Generated for enum switch.
+      "io.opentelemetry.auto.instrumentation.grpc.common.GrpcHelper$1",
```

Comment:
As far as I can tell, this mysterious class is being generated because of the switch on enum.